acdh-schema.owl

```diff
@@ -1729,6 +1788,22 @@ https://www.iana.org/assignments/character-sets/character-sets.xhtml</rdfs:comme
     
 
 
+    <!-- https://vocabs.acdh.oeaw.ac.at/schema#hasIiifManifest -->
+
+    <owl:DatatypeProperty rdf:about="https://vocabs.acdh.oeaw.ac.at/schema#hasIiifManifest">
+        <rdfs:subPropertyOf rdf:resource="https://vocabs.acdh.oeaw.ac.at/schema#disseminationServices"/>
+        <rdfs:domain rdf:resource="https://vocabs.acdh.oeaw.ac.at/schema#BinaryContent"/>
+        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#anyURI"/>
+        <rdfs:comment xml:lang="de">Gibt das IIIF-Manifest (B) an, das für die Darstellung der Ressource (A) verwendet werden soll. Der Wert ist entweder die URL des zu verwendenden IIIF-Manifests (externe URLs sind möglich, wir empfehlen jedoch Ressourcen aus dem Repositorium) oder, wenn das IIIF-Manifest automatisch auf der Grundlage der Metadaten der Ressource generiert werden soll, die URI des Dissemination-Services: https://id.acdh.oeaw.ac.at/dissemination/iiifmanifest.</rdfs:comment>
+        <rdfs:comment xml:lang="en">Indicates the IIIF manifest (B) to be used for the presentation of the Resource (A). The value is either an URL of the IIIF manifest to be used (we recommend it to be another Resource in the repository but it can also be external)  or, if the IIIF manifest should be generated automatically based on the Resoure&apos;s metadata, the URI of the dissemination service: https://id.acdh.oeaw.ac.at/dissemination/iiifmanifest.</rdfs:comment>
```

Comment:
Two small typos:
1. delete double white space between `external)` and `or`
2. correct `Resoure` to `Resource`